core/src/nucleus/ribosome/callback/validation_package.rs

```diff
@@ -39,9 +39,13 @@ pub fn get_validation_package_definition(
 
             let call = ZomeFnCall::new(
                 &zome_name,
-                None,
+                make_internal_capability_request(
```

Comment:
Same question as above. These `__hdk_*` calls should not need a capability token, imho (-> `DirectCall`)